pay_ios/ios/Classes/PaymentHandler.swift

```diff
@@ -213,172 +213,3 @@ extension PaymentHandler: PKPaymentAuthorizationControllerDelegate {
     }
   }
 }
-
-/// A set of utility methods associated to `PKPayment`.
-extension PKPayment {
-  
-  /// Creates a `Dictionary` representation of the `PKPayment` object.
-  public func toDictionary() -> [String: Any?] {
-    return [
-      "token": String(decoding: token.paymentData, as: UTF8.self),
-      "paymentMethod": token.paymentMethod.toDictionary(),
-      "billingContact": billingContact?.toDictionary(),
-      "shippingContact": shippingContact?.toDictionary(),
-      "shippingMethod": shippingMethod?.toDictionary(),
-    ]
-  }
-}
-
-/// A set of utility methods associated to `PKPaymentMethod`.
-extension PKPaymentMethod {
-  
-  /// Creates a `Dictionary` representation of the `PKPaymentMethod` object.
-  public func toDictionary() -> [String: Any?] {
-    return [
-      "displayName": displayName,
-      "network": network?.rawValue,
-      "type": type.rawValue
-    ]
-  }
-}
-
-/// A set of utility methods associated to `PKContact`.
-extension PKContact {
-  
-  /// Creates a `Dictionary` representation of the `PKContact` object.
-  public func toDictionary() -> [String: Any?] {
-    return [
-      "name": "\(name?.givenName ?? "") \(name?.familyName ?? "")",
```

Comment:
The reason I have removed this hardcoded concatenation is because it sounds kinda opinionated to me.

IMO, this plugin's job is to forward the data from PassKit back to Flutter, then the user decides if he wants to concat the first name and the last name.

This concatenation makes sense for most of the occidental countries but other countries may require the other parts of the name object ([PersonNameComponents](https://developer.apple.com/documentation/foundation/personnamecomponents#topics))

Reply:
That's a fair point.